packages/contractkit/src/providers/celo-provider.ts

```diff
@@ -1,3 +1,4 @@
+import { Lock } from '@celo/utils/lib/lock'
```

Comment:
think you need to bump utils version and publish to npm

Reply:
Do you know the most up to date process for this? When do I publish?

Reply:
[How to publish a new npm package
](https://github.com/celo-org/celo-monorepo/blob/f97e19fc1311dad65c1039ec97051f93fd7e937e/README-dev.md
)
IIUC, I think it's simpler since the references here are all suffixed with `-dev.` so just bumping them all in this rb and then publishing in npm should be fine.  there's only an issue if there's a mobile release and they remove the `dev` suffix before the npm gets published (which would result in a build failure for them).